site/content/docs/background/architecture.md

```diff
@@ -90,14 +92,13 @@ cleanly enable this integration.
 
 Pinniped supports the following cluster integration strategies.
 
-* Pinniped hosts a credential exchange API endpoint via a Kubernetes aggregated API server.
+* Token Credential Request API: Pinniped hosts a credential exchange API endpoint via a Kubernetes aggregated API server.
 This API returns a new cluster-specific credential using the cluster's signing keypair to
 issue short-lived cluster certificates. (In the future, when the Kubernetes CSR API
 provides a way to issue short-lived certificates, then the Pinniped credential exchange API
 will use that instead of using the cluster's signing keypair.)
-
-More cluster integration strategies are coming soon, which will allow Pinniped to
-support more Kubernetes cluster types.
+* Impersonation Proxy: Pinniped hosts an [impersonation](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#user-impersonation)
+proxy that sends requests to the Kubernetes API server with user information and permissions based on a token. 
 
 ## kubectl Integration
 
```

Comment:
In "Pinniped Deployment Strategies" below, there are a couple places[0] that talk about exchanging external credentials for "a credential that is understood by the host Kubernetes cluster". Should we change this language to be vague enough to include both the Kube API server and the impersonation proxy? Maybe "a credential that can be used to authenticate to the host Kubernetes cluster"? Meh, I typed that out and I don't love that because it equally sounds like we are talking about the actual Kube API server. Maybe "a credential that can be used to authenticate to Kubernetes cluster"? IDK.

[0] https://github.com/vmware-tanzu/pinniped/blame/main/site/content/docs/background/architecture.md#L119, https://github.com/vmware-tanzu/pinniped/blame/main/site/content/docs/background/architecture.md#L139

Reply:
Done.